### PR TITLE
♻️ Remove hardcoded CORS URLs

### PR DESCRIPTION
**Pull request by jackreimers**
Removes hardcoded URLs from code and prevents staging build being able to connect to prod and vice versa.

Related to #120

### CODE REVIEW

File: src/WebAPI/appsettings.Development.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+﻿{
+  "AllowedCORSOrigins": [
+    "https://localhost:5002"
+  ]
```
Comment (Harry-Ross) [inline src/WebAPI/appsettings.Development.json:4]: Isn't the format comma-separated strings?

File: src/WebAPI/DependencyInjection.cs [hunk 2]
```diff
@@ -32,29 +34,21 @@ public static class DependencyInjection
...
-
-        var developmentCorsUrls = new string[] { "https://localhost:5002" };
-
+        
+        var allowedCors = configuration.GetValue<string>("AllowedCORSOrigins");
+        if (allowedCors == null)
+            throw new ArgumentException("No CORS origins specified in configuration.");
+        
+        //Workaround for not being able to set arrays as variables in Azure
+        var allowedCorsList = allowedCors.Split(",");
```
Comment (Harry-Ross) [inline src/WebAPI/DependencyInjection.cs:43]: Is this clear to people modifying the string in Azure?

**Review by Harry-Ross — APPROVED** [review]
lgtm 🛥️

File: src/WebAPI/appsettings.Development.json (hunk 1 above)
Comment (jackreimers) [reply src/WebAPI/appsettings.Development.json]: Yeah, good catch. I've updated it

File: src/WebAPI/DependencyInjection.cs (hunk 2 above)
Comment (jackreimers) [reply src/WebAPI/DependencyInjection.cs:43]: Probably not but I'm not really sure how we can resolve that.
As part of #32 the value will be coming from GitHub anyway, perhaps we can put a note in the repo README?

**Pull request merged**